connection_integration_test.go

```diff
@@ -22,11 +22,6 @@ func TestConnectionSetStatement(t *testing.T) {
 	_, err = conn.QueryContext(context.TODO(), "SELECT * FROM information_schema.tables", nil)
 	assert(err == nil, t, "query returned an error, but shouldn't")
 
-	_, err = conn.ExecContext(context.TODO(), "SET use_standard_sql=0", nil)
-	assert(err == nil, t, "set use_standard_sql returned an error, but shouldn't")
-
-	_, err = conn.QueryContext(context.TODO(), "SELECT * FROM information_schema.tables", nil)
-	assert(err != nil, t, "query didn't return an error, but should")
```

Comment:
Do those not work anymore?

Reply:
No 😞 
![image](https://user-images.githubusercontent.com/10419887/203773887-7d651b35-3364-4be1-b3e0-e4c30920af16.png)